From c48f8498e72928778a0501f8c958f0f61c214858 Mon Sep 17 00:00:00 2001
From: Natanael Copa <ncopa@alpinelinux.org>
Date: Sat, 20 Apr 2024 08:16:55 +0200
Subject: [PATCH] abuild: allow packages to install under run

I have some vauge memory that it is conveinent for Docker users that the
state dir under /run exists.

Allow this for alpine 3.20 and revisit later.
---
 abuild.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/abuild.in b/abuild.in
index ac0b6ff..f873620 100644
--- a/abuild.in
+++ b/abuild.in
@@ -782,7 +782,7 @@ postcheck() {
 	msg "Running postcheck for $name"
 	# checking for FHS compat
 	if ! options_has "!fhs"; then
-		for i in srv usr/local opt run var/run tmp var/tmp var/lock var/empty home sys proc mnt dev; do
+		for i in srv usr/local opt var/run tmp var/tmp var/lock var/empty home sys proc mnt dev; do
 			for j in "$dir"/"$i"/* "$dir"/"$i"/.[!.]* "$dir"/"$i"/..?*; do
 				if [ -L "$j" ] || [ -e "$j" ]; then
 					error "Packages must not put anything under /$i"
-- 
2.44.0

